src/RpcClient/RpcClient.cs

```diff
@@ -459,6 +450,49 @@ public async Task<RpcUnclaimedGas> GetUnclaimedGasAsync(string address)
             return RpcUnclaimedGas.FromJson(result);
         }
 
+        /// <summary>
+        /// Returns all results from Iterator.
+        /// This RPC call does not affect the blockchain in any way.
+        /// </summary>
+        public async IAsyncEnumerable<JObject> TraverseIteratorAsync(string sessionId, string id, int count, bool fetchAll = false)
```

Comment:
What about remove `fetchAll`? If `count == -1`, we fetch all items.

Reply:
How to specify the rpc parameter "count" value in this scenario?

Reply:
`100` or something...